lib/ansible/modules/packaging/os/zypper_repository.py

```diff
@@ -52,6 +53,12 @@
         type: bool
         default: 'yes'
         aliases: [ "refresh" ]
+    extra_args_precommand:
+       version_added: "2.8"
+       required: false
+       description:
+         - Add additional global target options to C(zypper).
+         - Options should be supplied in a single line as if given in the command line.
```

Comment:
Please parameter type information to the DOCUMENTATION, this way the generated docs include types as well.
Also, we prefer to see first the *description*, then *type*, then *choices* and *default* and then *version_added*.
(Most important for docs at the top)
```suggestion
         - Options should be supplied in a single line as if given in the command line.
       type: str
       version_added: '2.8'
```

Reply:
If we don't add type information the docs don't look as good:
https://docs.ansible.com/ansible/devel/modules/zypper_repository_module.html

Reply:
Thanks and Done